### PR TITLE
use official images

### PR DESCRIPTION
**Pull request by paulfantom**
SSIA

/assign @enxebre @ingvagabund

### CODE REVIEW

**Comment by bison** [comment]
My worry is that these tags seem to be overwritten with each build of master. Is that wrong?

What are other second-level-operators doing? Should the image and / or tag be a parameter in the config?

File: manifests/clusterapi-controller.yaml [hunk 1]
```diff
@@ -49,10 +49,10 @@ spec:
             memory: 30Mi
       {{- if .AWS }}
       - name: aws-machine-controller
-        image: openshift/origin-machine-controller:v4.0.0
+        image: openshift/origin-aws-machine-controllers:4.0.0
```
Comment (ingvagabund) [inline manifests/clusterapi-controller.yaml]: https://hub.docker.com/r/openshift/origin-aws-machine-controllers/

File: manifests/clusterapi-controller.yaml [hunk 2]
```diff
@@ -49,10 +49,10 @@ spec:
...
       {{- else if .Libvirt}}
       - name: libvirt-machine-controller
-        image: quay.io/coreos/cluster-api-provider-libvirt:cd386e4 # TODO: move this to openshift org
+        image: openshift/origin-libvirt-machine-controllers:4.0.0
```
Comment (ingvagabund) [inline manifests/clusterapi-controller.yaml]: https://hub.docker.com/r/openshift/origin-libvirt-machine-controllers/

**Comment by ingvagabund** [comment]
> My worry is that these tags seem to be overwritten with each build of master. Is that wrong?

> What are other second-level-operators doing? Should the image and / or tag be a parameter in the config?

We still don't have any releases. So for time being it's fine to take the latest (a.k.a 4.0.0). Though, in the future the image version needs to be made configurable.

**Comment by enxebre** [comment]
this is in now

**Pull request closed without merging**